src/server/login/login-middleware.js

```diff
@@ -141,6 +141,7 @@ const setup = (app: Router) => {
     '/auth/ping',
     requestRateLimiter(500),
     wrapAsync(async (req, res) => {
+      req.log.error('qqq', 'second', new Error('third error'), { test: true })
```

Comment:
```suggestion
      req.log.error('qqq', 'second', new Error('third error'), { test: true })
```

please, remove test logs / other similar stuff before commit